app/javascript/chat/RequestManager/RequestListItem.jsx

```diff
@@ -0,0 +1,55 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+const RequestListItem = ({
+  request,
+  handleRequestRejection,
+  handleRequestApproval,
+}) => (
+  <div className="crayons-card mb-6">
+    <div className="crayons-card__body channel-request-card">
+      <div className="request-message d-flex flex-wrap">
+        <b>{request.name}</b> wanted to join <b>{request.chat_channel_name}</b>
```

Comment:
Suggestion: Replace "wanted" with "requested" for consistency (users "request" to join channels)